Cargo.toml

```diff
@@ -2,7 +2,7 @@
 version = "0.5.43"                                        # same as `hotshot`, but workspace subcrate can also release its own version
 authors = ["Espresso Systems <hello@espressosys.com>"]
 edition = "2021"
-rust-version = "1.65.0"
+rust-version = "1.76.0"
```

Comment:
Why do we want to update this? As far as I can tell everything still builds with 1.65

Reply:
It does not in fact build on 1.65, sorry! These changes are correct.